src/app/header/header.component.scss

```diff
@@ -0,0 +1,54 @@
+@import “../variables”;
+
+.header-bar {
+  background: $header-background;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.toggle-button {
+  background-color: $header-background;
+  border-width: 0;
+  &:hover {
+    background-color: $button-hover-color;
+    border-width: 0;
+	}
+	&:active {
+    background-color: $header-background;
+    border-width: 0;
+	}
+}
+
+.header-1 {
+	color: $headerbar-header-left-color;
+	font-weight: $header-font-weight;
+	font-size: $header-font-size;
+}
+
+.header-2 {
+	color: $header-right-colors;
+	font-weight: $header-font-weight;
+	font-size: $header-font-size;
+}
+
+.logout-button {
+	background-color: $header-background;
+	border-width: 0;
+	float: right;
+	color: #FFFFFF;
```

Comment:
yes but those variables are for header background, if someone were to change the header color variable for changing color of header, it would also change the logout button's font color.  Will add new variable for this.